Lesson-4-Task-2/src/main/java/Lesson4/Task2/Triangle.java

```diff
@@ -0,0 +1,22 @@
+package Lesson4.Task2;
+import Lesson4.Shape;
+public class Triangle extends Shape {
+     int a;
+     int h;
+     int b;
+     int c;
+    Triangle(int sideA, int sideB, int sideC, int height){
```

Comment:
default scope?

Reply:
Przyjołem zakres z klasy Triangle. Zmieniłem zmienne a,height,b,c na private. Triangle zostawiłem na zakres default czyli po classie Triangle -  public.

Reply:
Zakres default nie sprawi, że konkstruktor będzie public dlaczego, że klasa tak ma. 
Brak zasięgu to zasięg package-private.
Daj publica ;)